### DISCUSSION TITLE
[jsvm] How to detect PocketBase version?

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#6006] [Q&A] [answered]
I'm looking for something like `$app.version()` that returns `0.23.4` or whatever. Is this available?

**Comment by ganigeorgiev** [comment, marked as answer]
You can try with `$app.rootCmd.version` but note that this by default is set to the tag version only for the prebuilt executables as part of the [goreleaser build command](https://github.com/pocketbase/pocketbase/blob/master/.goreleaser.yaml#L16).
